src/ui/components/GeneralSettings.tsx

```diff
@@ -226,25 +223,44 @@ export const GeneralSettings = () => {
         )}
       </div>
       <Separator />
-      <div className="grid-2-col">
-        <div>
+      <div>
           <Title size="small" weight="bold">
-            Reference mode in variables
-            <Info
-              width={240}
-              label="If disabled, the exported json will not include the mode of variables."
-            />
+            Reference mode from variables
           </Title>
-          <Checkbox
-            label="Enable/disable mode referencing"
-            type="switch"
-            checked={settings.modeReference}
-            onChange={(value) =>
-              updateSettings((draft) => {
-                draft.modeReference = value;
-              })
-            }
-          />
+        <div className="grid-2-col">
+          <div>
+            <Checkbox
+              label="Add mode to design token name"
```

Comment:
```suggestion
              label="Add mode to design token name (if 2 or more modes)"
```

Reply:
I have updated this one, thanks for the suggestion.